core/OpenStore365.Serverless/HTTP.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace OpenStore365.Serverless
+{
+    public static class HTTP
+    {
+        public const string GET = nameof(GET);
```

Comment:
Cóż to takiego?

Reply:
I don't like to have literal strings in my `[HttpTrigger]`. I swap them out with strongly typed alternative:
`[HttpTrigger(AuthorizationLevel.Anonymous, "get", Route = "Http/Headers/UserClaims")] ` 
`[HttpTrigger(AuthorizationLevel.Anonymous, HTTP.GET, Route = "Http/Headers/UserClaims")] `